### DISCUSSION TITLE
Node Upgrades don't suggest upgrades for non-LTS Node

### DISCUSSION THREAD

**Discussion by broksonic21** [renovatebot/renovate#21973] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

[Node 20.1.0](https://nodejs.org/en/blog/release/v20.1.0) is offered as a PR

### What you tried so far.

See [https://github.com/broksonic21/node-non-lts-renovate-bot](https://github.com/broksonic21/node-non-lts-renovate-bot)  as a sample - this has node 20.0.0 in both the .nvmrc and the Volta section of package.json

In private repositories, when I used to be on LTS (Node 18), I got renovate PRs for Node upgrades. But once we switched to node 20, RenovateBot no longer notices 20.1.0 as an update so isn't offering it.

### Relevant debug logs

<details><summary>Logs</summary>

```
DEBUG: packageFiles with updates
{
  "baseBranch": "main",
  "config": {
    "npm": [
      {
        "constraints": {},
        "deps": [
          {
            "commitMessageTopic": "Node.js",
            "currentValue": "20.0.0",
            "currentVersion": "20.0.0",
            "datasource": "github-tags",
            "depName": "node",
            "depType": "volta",
            "fixedVersion": "20.0.0",
            "lockedVersion": null,
            "packageName": "nodejs/node",
            "prettyDepType": "volta",
            "registryUrl": "https://github.com",
            "sourceUrl": "https://github.com/nodejs/node",
            "versioning": "node",
            "warnings": [],
            "updates": []
          },
          {
            "currentValue": "9.6.6",
            "currentVersion": "9.6.6",
            "datasource": "npm",
            "depName": "npm",
            "depType": "volta",
            "fixedVersion": "9.6.6",
            "homepage": "https://docs.npmjs.com/",
            "lockedVersion": null,
            "packageName": "npm",
            "prettyDepType": "volta",
            "registryUrl": "https://registry.npmjs.org",
            "sourceUrl": "https://github.com/npm/cli",
            "versioning": "npm",
            "warnings": [],
            "updates": []
          }
        ],
        "lockFiles": [
          "package-lock.json"
        ],
        "managerData": {
          "hasPackageManager": false,
          "npmLock": "package-lock.json",
          "packageJsonName": "node-non-lts-renovate-bot",
          "yarnZeroInstall": false
        },
        "packageFile": "package.json",
        "packageFileVersion": "0.0.2",
        "skipInstalls": true
      }
    ],
    "nvm": [
      {
        "deps": [
          {
            "currentValue": "20.0.0",
            "currentVersion": "20.0.0",
            "datasource": "github-tags",
            "depName": "node",
            "fixedVersion": "20.0.0",
            "packageName": "nodejs/node",
            "registryUrl": "https://github.com",
            "sourceUrl": "https://github.com/nodejs/node",
            "versioning": "node",
            "warnings": [],
            "updates": []
          }
        ],
        "packageFile": ".nvmrc"
      }
    ]
  }
}
```

</details>

**Comment by rarkins** [comment]
Set ignoreUnstable=false for node in a package rule

**Comment by broksonic21** [comment]
Thanks @rarkins . Can you explain why this is unstable?  reading the docs: https://docs.renovatebot.com/configuration-options/#ignoreunstable

It talks about -rc, -alpha, etc. But 20.1.0 wouldn't qualify as either of those, right? Or is node handled in a more special/unique manner from other packages for non-LTS? If so, can either https://docs.renovatebot.com/configuration-options/#ignoreunstable or https://docs.renovatebot.com/node/ (preferably) be updated to explain this?

**Reply by rarkins** [reply]
When distributions have a concept of LTS, we treat non-LTS as unstable
